mk/db.mk

```diff
@@ -0,0 +1,38 @@
+##
+# Set of rules to interact with a local database 
+# from a container and database initialization.
+##
+
+.PHONY: db-up
+db-up: DOCKER_IMAGE=docker.io/postgres:14
+db-up: $(GO_OUTPUT)/dbmigrate ## Start postgres database
+	docker volume exists postgres || docker volume create postgres
+	docker container exists postgres || docker run \
```

Comment:
can we replace all these with $(DOCKER)  :)

Reply:
sure! updated! :)